cli/internal/runsummary/format_json.go

```diff
@@ -21,19 +17,18 @@ func (rsm *Meta) FormatJSON() ([]byte, error) {
 	return bytes, nil
 }
 
-func (rsm *Meta) formatJSONSinglePackage() ([]byte, error) {
-	singlePackageTasks := make([]singlePackageTaskSummary, len(rsm.RunSummary.Tasks))
-
-	for i, task := range rsm.RunSummary.Tasks {
-		singlePackageTasks[i] = task.toSinglePackageTask()
+func (rsm *Meta) normalize() {
```

Comment:
`normalize()` is the thing that mutates the RunSummary, we are already doing this for env vars, so it was a natural fit.